app/views/products/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<h2>Edit a Product</h2>
+<%= form_for @product, method: :patch, url: vendor_product_path(@vendor, @product) do |f| %>
+
+<%= f.label :'Product Name:' %>
+	<br>
+	<%= f.text_field :name, class: 'field' %>
+	<br>
+	<%= f.submit %>
+<% end %>
```

Comment:
Github is complaining that you don't have a newline at the end of some of your files. If you're using Atom and you hit save, it automatically adds a newline. I'm not sure why this matters.